### PR TITLE
Scores branch

### PR DESCRIPTION
**Pull request by nayyyhaa**
## In this PR :
- I've added the scores page for auth users.

**ONLY check the following files**

#### components: 
- [x] [components/LoginForm.tsx](https://github.com/nayyyhaa/innotion-quiz-react/pull/5/files#diff-ee11ef59f37648b9c892365426b8d0abc519b0aee7f0e7ac78e44a5844d62280)
- [x] [components/Navbar.tsx](https://github.com/nayyyhaa/innotion-quiz-react/pull/5/files#diff-a0439d8a2cb6dee715c41a795bb174a2a65ad61704e88691fe5ab2b2bd6f798e)

#### pages: 
- [x] [pages/ResultPage.tsx](https://github.com/nayyyhaa/innotion-quiz-react/pull/5/files#diff-73426a0e72301264c7210082d12075fc4bb8d0d0cb2e1c89cb314ebf5efa5ab7)
- [x] [pages/ScoresPage.tsx](https://github.com/nayyyhaa/innotion-quiz-react/pull/5/files#diff-7c3c5d249c744a8fb3e9f631bfaf82980c78e5d2d92801035f281a1bae759d43)

#### others:
- [x] [contexts/QuizContext.tsx](https://github.com/nayyyhaa/innotion-quiz-react/pull/5/files#diff-3c63fdd4e1bd6ce3f240f52032cf931a7f91475e93b6caaaa6fe8e7d10f6a09f)
- [x] [utils/formatDate.tsx](https://github.com/nayyyhaa/innotion-quiz-react/pull/5/files#diff-669f7fedd8b31204461232ddff923e85d652de69aa08d7abf10d77c67ebc7290)

#### files to *ignore*
- [ ] data files
- [ ] CSS files
- [ ] index files
- [ ] backend files

### CODE REVIEW

File: src/pages/ResultPage.tsx [hunk 1]
```diff
@@ -1,25 +1,46 @@
...
   return (
     <main className="form-content full-wd grid-ctr m-auto p-h-2 p-v-5">
-      <h2 className="title colored-text m-v-1">
-        <span className="circle"></span>What is Mindfulness?
+      <h2 className="title centered-text colored-text m-v-1">
+        <span className="circle"></span>
+        {currentQuiz?.title}
       </h2>
-      <h3 className="h2 m-v-2">Final Score: {totalScore}/10</h3>
+      {/* <button onClick={add}>AD</button> */}
```
Comment (hsnice16) [inline src/pages/ResultPage.tsx]: If this is not required, you can remove the commented code.

**Review by hsnice16 — APPROVED** [review]
Really great work. Couldn't find much to suggest. Good Job.

**Review by Developers-codz — APPROVED** [review]
Good work Neha, both UI and code look clean, you can merge

File: src/pages/ResultPage.tsx (hunk 1 above)
Comment (nayyyhaa) [reply src/pages/ResultPage.tsx]: Done with the changes.

**Comment by nayyyhaa** [comment]
Thanks for the review. Merging the changes.

**Pull request merged**